### PR TITLE
POC - docs site sitemap.xml

### PR DESCRIPTION
**Pull request by jamieomaguire**
Please click the `Preview` tab and select a different PR template if your change isn't for an                                            existing web component.

[PIE Docs Template](?expand=1&template=docs_template.md)
[New Web Component Template](?expand=1&template=new_component_template.md)

═══════════════════════════════════════════════════════════

## Author Checklist (complete before requesting a review)
- [ ] I have performed a self-review of my code
- [ ] I have added thorough tests where applicable (unit / component / visual)
- [ ] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [ ] I have reviewed visual test updates properly before approving
- [ ] If changes will affect consumers of the package, I have created a changeset entry.
- [ ] If a changeset file has been created, I have used the `/snapit` functionality to test my changes in a consuming application

### Reviewer 1
- [ ] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [ ] If there are visual test updates, I have reviewed them

### Reviewer 2
- [ ] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [ ] If there are visual test updates, I have reviewed them

### CODE REVIEW

**Comment by xander-marjoram** [comment]
Ordinary comment

**Review by xander-marjoram — COMMENTED** [review]
Review comment

File: apps/pie-docs/src/_data/projectSettings.js [hunk 1]
```diff
@@ -2,5 +2,6 @@
 module.exports = function () {
     return {
         environment: process.env.DEPLOYMENT_ENVIRONMENT || 'development',
+        baseUrl: process.env.DEPLOYMENT_ENVIRONMENT === 'production' ? 'https://www.pie.design' : '',
```
Comment (xander-marjoram) [inline apps/pie-docs/src/_data/projectSettings.js:5]: Inline comment

Comment (jamieomaguire) [reply apps/pie-docs/src/_data/projectSettings.js:5]: nice, @xander-marjoram testing

Comment (xander-marjoram) [reply apps/pie-docs/src/_data/projectSettings.js:5]: Replying without tag

Comment (xander-marjoram) [reply apps/pie-docs/src/_data/projectSettings.js:5]: @jamieomaguire replying with tag

**Pull request closed without merging**